### PR TITLE
todo list @ngfizzy

### PR DESCRIPTION
**Pull request by Nettar-Jr**
... @ngfizzy

### CODE REVIEW

**Comment by Nettar-Jr** [comment]
... @ngfizzy

File: todo.js [hunk 1]
```diff
@@ -1,104 +1,48 @@
...
-		storeItems();
-	 
-		
+	if (inputValue === ''){
+		alert ('please enter a task');
+	}
+	else {
+		document.getElementById('list').appendChild(todoItem);
+	}
+    document.getElementById('input').value;
```
Comment (ngfizzy) [inline todo.js:30]: fix indentation here. line 30 does not align well with  the if else statement above

File: todo.js [hunk 2]
```diff
@@ -1,104 +1,48 @@
...
+     
 	// hide list when delete button is clicked
-	deleteButton.onclick = function(event){
-		event.stopPropagation();
-		todoItem.parentNode.removeChild(todoItem);	
-		
-		storeItems();
+	 
+	deleteButton.onclick = function(){
+		todoItem.style.display = "none";
```
Comment (ngfizzy) [inline todo.js:35]: why are you hiding the element instead of actually removing the node?

**Review by ngfizzy — CHANGES_REQUESTED** [review]
Nice one. I love the little animation that you added. Spices things up a lil bit.

I also noticed that you cleaned up your code a lot more. Nice one.

Few feedback.

There are still quite a number of features yet to be implemented.
Let me know if you'd be needing a pairing session so we can figure this out. Nice job once again

**Pull request merged**